src/guide/essentials/template-refs.md

```diff
@@ -234,14 +235,14 @@ defineExpose({
 </script>
 ```
 
-When a parent gets an instance of this component via template refs, the retrieved instance will be of the shape `{ a: number, b: number }` (refs are automatically unwrapped just like on normal instances).
+Ota component template reflar orqali shu komponent namunasini olganda, olingan namuna `{ a: number, b: number }` ko'rinishda bo'ladi (reflar normal namunadagidek avtomatik o'ralmagan holda bo'ladi).
 
-See also: [Typing Component Template Refs](/guide/typescript/composition-api.html#typing-component-template-refs) <sup class="vt-badge ts" />
+Buni ham ko'ring: [Typing Component Template Refs](/guide/typescript/composition-api.html#typing-component-template-refs) <sup class="vt-badge ts" />
 
 </div>
 <div class="options-api">
 
-The `expose` option can be used to limit the access to a child instance:
+`expose` varianti(option)dan bola namuna bog'lanish - ulanishni cheklash uchun foydalanish mumkin:
```

Comment:
bu yerga ham probel qo'shish kerak varianti va qavs ( lar o'rtasida
![image](https://user-images.githubusercontent.com/12369295/172148042-8c11ac89-57cd-4651-9845-1005542e5eb3.png)

Reply:
Tushunarli.

Reply:
To'g'rilandi.